app/models/manageiq/providers/amazon/cloud_manager/event_catcher/stream.rb

```diff
@@ -110,19 +110,23 @@ def sqs_get_queue_url(queue_name)
     end
   end
 
-  # @return [Aws::SNS::Topic] the found topic
-  # @raise [ProviderUnreachable] in case the topic is not found
   def sns_topic
     @ems.with_provider_connection(:service => :SNS) do |sns|
-      sns.topics.detect { |t| t.arn.split(/:/)[-1] == @topic_name }
-    end || begin
-      $aws_log.warn("#{log_header} Unable to find the AWS Config Topic '#{@topic_name}'. " \
-      "Cannot collect Amazon events for AWS Access Key ID #{@ems.authentication_userid}")
-      $aws_log.warn("#{log_header} Contact Amazon to create the AWS Config service and topic for Amazon events.")
-      raise ProviderUnreachable, "Unable to find the AWS Config Topic '#{@topic_name}'"
+      get_topic(sns) || create_topic(sns)
     end
   end
 
+  def get_topic(sns)
+    sns.topics.detect { |t| t.arn.split(/:/)[-1] == @topic_name }
+  end
+
+  def create_topic(sns)
+    sns.create_topic(:name => @topic_name)
```

Comment:
to enable aws config, you have to log into the aws console and enable it.

Now what happens, if the use has AWS Config turned off? Or not initially turned it on.
Before he could see in the logs, that he should enable config.
Now he sees nothing, because we just listen to a SQS queue that is subscribed to a SNS topic, that nobody sends to.

Is that right? Or am I missing something?

<img width="796" alt="screen shot 2017-02-07 at 17 51 14" src="https://cloud.githubusercontent.com/assets/161888/22701492/0f9a09e6-ed5e-11e6-9821-9de5c84faa00.png">